homeassistant/components/tplink_omada/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/tplink_omada",
   "integration_type": "hub",
   "iot_class": "local_polling",
-  "requirements": ["tplink-omada-client==1.4.2"]
+  "requirements": ["tplink-omada-client==1.4.3"]
```

Comment:
The dependency bump should be split into a separate PR.

Could you please do that?

Thanks! 👍 

../Frenck

Reply:
Thanks for the feedback, did as requested in: https://github.com/home-assistant/core/pull/130184